api/v3/Case.php

```diff
@@ -240,6 +240,22 @@ function civicrm_api3_case_get($params) {
     $sql->where("a.id IN (SELECT case_id FROM civicrm_case_contact WHERE $clause)");
   }
 
+  // Order by case contact (primary client)
```

Comment:
At first I was little perplexed trying to understand why this was doing all the explodes and pads and such. It took a bit of reading/thinking, and then reading the test confirmed the theory.

The comment -- in the code and/or in the commit message -- could do a better job of framing the code, maybe:

```php
// Order by case-contact properties. Ex: `contact_id`, `contact_id.display_name`, `contact_id.sort_name DESC`.
```